cmd/kubeadm/app/preflight/checks.go

```diff
@@ -607,3 +607,20 @@ func TryStartKubelet() {
 		}
 	}
 }
+
+// RunPreFlightChecks runs all pre-flight checks
+func RunPreFlightChecks(cfg *kubeadmapi.MasterConfiguration) error {
+	fmt.Println("[preflight] Running pre-flight checks")
+
+	// First, check if we're root separately from the other preflight checks and fail fast
+	if err := RunRootCheckOnly(); err != nil {
```

Comment:
I came to think about that you could make `RunInitMasterChecks` (and for join) call `RunRootCheckOnly`, and we'd only have one method to call. I think that would be a good improvement